script/ide/System/Core/ToolBase_p.lua

```diff
@@ -65,7 +65,11 @@ function ConnectionSynapse:Activate(...)
 		if(self.receiver) then
 			self.receiver.currentSender = self.sender;
 		end
-		self.slot(self.receiver, ...);
+		if(self.receiver) then
```

Comment:
are you sure about this changes?  I think if self.receiver is nil, it should be self.slot(nil, ...).  
is it better to remain the old way?